py/desitarget/mock/QA.py

```diff
@@ -0,0 +1,220 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+desitarget.mock.QA
+==================
+
+Generate QA figures from the output of desitarget.mock.join_targets_truth.
+
+"""
+from __future__ import (absolute_import, division, print_function)
+
+import os
+import numpy as np
+import warnings
+
+def target_density(cat):
```

Comment:
pls update docstrings to define inputs.  e.g. here, that `cat` is a table with columns RA and DEC.